src/pymor/vectorarrays/interfaces.py

```diff
@@ -58,9 +58,12 @@ class VectorArrayInterface(BasicInterface):
         `True` if the array is a view obtained by indexing another array.
     space
         The |VectorSpace| the array belongs to.
+    dtype
+        Field data type of the contained vectors.
```

Comment:
We still need a more precise definition here. I can write one, once we have agreed on the right semantics. In particular, I think we really should define, what the `dtype` of the norm functions should be. Probably the real version of the array's dtype?

Reply:
OK